### PR TITLE
Update input provider readme

### PR DESCRIPTION
**Pull request by thekp**
Resolves #NUMBER

**Overall change:** _i missed out updating this section of the readme in my initial refactor PR_

code changes:

- _the readme needed updating to say input provider accepts a single object and describe what props do_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-storybook-helpers/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.1 | [PR#1926](https://github.com/bbc/psammead/pull/1926) Update readme with changes to input provider |
```
Comment (ghost) [inline packages/utilities/psammead-storybook-helpers/CHANGELOG.md]:
```suggestion
| 6.0.1 | [PR#2042](https://github.com/bbc/psammead/pull/1926) Update readme with changes to input provider |
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 2]
```diff
@@ -6,18 +6,20 @@ This package provides a collection of common values that are used in storybook b
 
 `LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.
 
-`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes two arguments, `slots` and `renderFn`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `renderFn`. This should usually be a React Component.
+`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 props, `slots` and `componentFunction`, `services` and `options`.Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```
Comment (ghost) [inline packages/utilities/psammead-storybook-helpers/README.md]:
```suggestion
`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`.Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 3]
```diff
@@ -6,18 +6,20 @@ This package provides a collection of common values that are used in storybook b
 
 `LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.
 
-`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes two arguments, `slots` and `renderFn`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `renderFn`. This should usually be a React Component.
+`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`.Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```
Comment (thekp) [inline packages/utilities/psammead-storybook-helpers/README.md]:
```suggestion
`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```

**Review by FK78 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: packages/utilities/psammead-storybook-helpers/README.md [hunk 4]
```diff
@@ -6,18 +6,20 @@ This package provides a collection of common values that are used in storybook b
 
 `LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.
 
-`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes two arguments, `slots` and `renderFn`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `renderFn`. This should usually be a React Component.
+`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```
Comment (dr3) [inline packages/utilities/psammead-storybook-helpers/README.md]: >Takes 4 arguments

Really it takes 1 arguement, an object with 4 possible keys 🙈

Comment (thekp) [inline packages/utilities/psammead-storybook-helpers/README.md]:
```suggestion
`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 1 argument of an object, with these possible keys: `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```

**Review by dr3 — APPROVED** [review]

**Review by oluoluoxenfree — APPROVED** [review]

File: packages/utilities/psammead-storybook-helpers/README.md (hunk 4 above)
Comment (ghost) [reply packages/utilities/psammead-storybook-helpers/README.md]: Good spot @dr3

**Pull request merged**